packages/generator/templates/mutations/create__ModelName__.ts

```diff
@@ -4,12 +4,12 @@ import * as z from "zod"
 
 if (process.env.parentModel) {
   const Create__ModelName__ = z.object({
-    name: z.string(),
+    /* template: __attributeName__: z.__zodTypeName__(), */
```

Comment:
Can we use `fieldName` here instead of `attributeName` to be consistent?

Also I think we can shorten `zodTypeName` to `zodType`